spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -1558,6 +1558,19 @@ If any `ProcessProducer` method is called outside of the observer method invocat
 
 CDI 2.0 introduced the `javax.enterprise.inject.spi.configurator.ProducerConfigurator` interface to help configuring a `Producer` instance.
 
+[source, java]
+----
+public interface ProducerConfigurator<T> {
+
+    <U extends T> ProducerConfigurator<T> produceWith(Function<CreationalContext<U>, U> callback);
+    ProducerConfigurator<T> disposeWith(Consumer<T> callback);
+    ProducerConfigurator<T> addInjectionPoint(InjectionPoint injectionPoint);
+    ProducerConfigurator<T> addInjectionPoints(InjectionPoint... injectionPoints);
+    ProducerConfigurator<T> addInjectionPoints(Set<InjectionPoint> injectionPoints);
+    ProducerConfigurator<T> injectionPoints(InjectionPoint... injectionPoints);
+}
+----
```

Comment:
-1 to remove them from the spec doc, we need to have them officially specified. Javadoc is an helper but is not officially art of the spec.